docs/user_guide/migrating_versions/migrating_to_v0_5_0.rst

```diff
@@ -10,15 +10,15 @@ Metrics
 -------
 
 We have substantially altered the :mod:`fairlearn.metrics` module.
-In place of calling ``group_summary()`` to produce a :py:class:`sklearn.utils.Bunch`
+In place of calling ``group_summary()`` to produce a :obj:`sklearn.utils.Bunch`
```

Comment:
Hmmm... I thought I'd checked that the right link was generated :-(

Reply:
Plenty of warnings in the output. This both works and there are no warnings so I replaced it.